src/chronos/i18n/pt-br.ts

```diff
@@ -12,8 +12,8 @@ import { getDayOfWeek } from '../units/day-of-week';
 
 export const ptBrLocale: LocaleData = {
   abbr: 'pt-br',
-  months: 'janeiro_fevereiro_março_abril_maio_junho_julho_agosto_setembro_outubro_novembro_dezembro'.split('_'),
-  monthsShort: 'jan_fev_mar_abr_mai_jun_jul_ago_set_out_nov_dez'.split('_'),
+  months: 'Janeiro_Fevereiro_Março_Abril_Maio_Junho_Julho_Agosto_Setembro_Outubro_Novembro_Dezembro'.split('_'),
```

Comment:
this localizations are compatible with https://github.com/moment/moment/blob/develop/src/locale/pt-br.js
so I can't merge such PRs
What you can do is update your locale in your app